docs/whats-new/release-candidate-1.md

```diff
@@ -0,0 +1,786 @@
++    <PackageReference Include="Aspire.Hosting.AppHost" Version="9.0.0-rc.1.24511.1" />
+  </ItemGroup>
+
+</Project>
+```
+
+That's it! You have successfully upgraded your project to .NET Aspire 9.0 RC 1. If you are using Visual Studio, it is recommended to upgrade to version 17.12 Preview 3 or later in order to take advantage of the new features and improvements for .NET Aspire 9.0.
+
+### Upgrade an existing project to .NET Aspire 9.0 using the Upgrade Assistant
+
+For this release, we are introducing .NET Aspire support to the Upgrade Assistant. The Upgrade Assistant is a tool that helps you upgrade your .NET projects to the latest .NET version. Now you can also use the Upgrade Assistant to upgrade your .NET Aspire projects to 9.0 RC 1.
+
+To start the upgrade process, right-click your AppHost project in Visual Studio and select the option `Upgrade`.
+
+![Upgrade using upgrade assistant](images/upgrade-assistant-1.png)
+
+If you don't yet have the Upgrade Assistant installed, then Visual Studio will guide you to the steps of how to install the extension. Once you have it installed, you will see the following option when you select the `Upgrade` option on your AppHost project:
+
+![Upgrade assistant options](images/upgrade-assistant-2.png)
+
+This will then do some analysis in your project and provide you with a report similar to the following detailing what will be updated. All you need to do now is to click the `Upgrade Selection` button to start the upgrade process:
+
+![Upgrade confirmation](images/upgrade-assistant-confirmation.png)
+
+That's it! You have successfully upgraded your project to .NET Aspire 9.0 RC 1 using the Upgrade Assistant.
+
+## Dashboard
+
+### Manage resource lifecycle
+
+The most requested feature for the dashboard is to manage the lifecycles of your orchestrated named resources. Specifically, the ability to stop, start, and restart resources. This feature is now available in .NET Aspire 9.
+
+<!-- markdownlint-disable MD033 -->
+<video controls src="https://github.com/user-attachments/assets/3937ca4e-4ac9-4af7-938d-d016aa6277dc" title="Demo of managing resource lifecycle in the dashboard"></video>
+<!-- markdownlint-enable MD033 -->
+
+This feature works for projects, containers and executables. It enables restarting indidivual resources without having to restart the entire app host. For projects, if the debugger is attached, on restart, the debugger will be re-attached.
+
+- Responsive design
+- Mobile friendly
+
+### Sensitive properties, volumes and health checks in resource details
+
+Resource details has seen a lot of improvements:
+
+- Properties can be marked as sensitive, automatically masking them in the dashboard UI. This is a security feature to avoid accidently disclosing keys or passwords when screen sharing the dashboard with other people. For example, container arguments could pass sensitive information and so are masked by default.
```

Comment:
```suggestion
- Properties can be marked as sensitive, automatically masking them in the dashboard UI. This security feature helps to avoid accidently disclosing keys or passwords when screen sharing the dashboard with other people. For example, container arguments could pass sensitive information and so are masked by default.
```